fuse/examples/hello.rs

```diff
@@ -1,53 +1,58 @@
 extern crate env_logger;
+#[macro_use]
+extern crate lazy_static;
 extern crate fuse;
 extern crate libc;
-extern crate time;
 
 use std::env;
 use std::ffi::OsStr;
+use std::time::{Duration, SystemTime, UNIX_EPOCH};
 use libc::ENOENT;
-use time::Timespec;
 use fuse::{FileType, FileAttr, Filesystem, Request, ReplyData, ReplyEntry, ReplyAttr, ReplyDirectory};
 
-const TTL: Timespec = Timespec { sec: 1, nsec: 0 };                     // 1 second
+lazy_static! {
+    // 1 second
+    static ref TTL: SystemTime = UNIX_EPOCH + Duration::new(1, 0);
```

Comment:
This feels strange, since it's setting a `SystemTime` to 1 second after the `UNIX_EPOCH`.
I feel like the `TTL` values should actually be `Duration`s rather than timestamps, is that right? Or have I misunderstood how this is meant to work?

Reply:
Indeed `Duration` makes much more sense here. I'm going to add a commit that replaces TTL types with `Duration`. Thanks for the hint!